src/interface/src/app/plan/plan-summary/summary-panel/summary-panel.component.ts

```diff
@@ -70,4 +81,17 @@ export class SummaryPanelComponent implements OnChanges {
       };
     }
   }
+
+  autoSaveNotes(): void {
+    interval(NOTE_SAVE_INTERVAL)
+      .pipe(untilDestroyed(this))
+      .subscribe(() => {
+        if (this.plan && this.plan.notes !== this.notes) {
+          this.plan.notes = this.notes;
+          this.planService
+            .updatePlanningArea(this.plan, this.plan.id)
+            .subscribe();
```

Comment:
I think that in order for the subscription to be terminated (with `untilDestroyed`)  you need to use a `switchMap` so you can be chained to the first subscription

Reply:
Something along this lines:
```
interval(NOTE_SAVE_INTERVAL)
      .pipe(
        untilDestroyed(this),
        filter((_) => this.plan && this.plan.notes !== this.notes),
        tap((_) => (this.plan.notes = this.notes)),
        switchMap((_) =>
          this.planService.updatePlanningArea(this.plan, this.plan.id)
        )
      )
      .subscribe();
```